engine/runtime/src/main/scala/org/enso/compiler/pass/resolve/TypeSignatures.scala

```diff
@@ -86,6 +86,11 @@ case object TypeSignatures extends IRPass {
         res
       case meth: IR.Module.Scope.Definition.Method =>
         val newMethod = meth.mapExpressions(resolveExpression)
+        val arr = newMethod.body match {
+          case fn: IR.Function => verifyAscribedArguments(fn.arguments)
+          case _               => newMethod
+        }
+        arr.getClass()
```

Comment:
?

Reply:
Workaround for annoying _"Unused variable warning"_ that fails Scala compilation. Left over some experiments.